src/main/java/edu/umich/ctools/sectionsUtilityTool/Utils.java

```diff
@@ -65,11 +65,11 @@ public static void openFile(HttpServletRequest request,
 
 		//CANVAS_API_SEARCH_USER
 		if (url.matches(SectionsUtilityToolServlet.appExtPropertiesFile.getProperty(CANVAS_API_SEARCH_USER))){
-			fr = retrieveTestFile(url, fr, "Courses call stub", "/stubs/canvas/users.txt");
+			fr = retrieveTestFile(url, fr, "Users call stub", "/stubs/canvas/users.txt");
 		}
 		
 		else if (url.matches(SectionsUtilityToolServlet.appExtPropertiesFile.getProperty(CANVAS_API_TERMS))){
-			fr = retrieveTestFile(url, fr, "Courses call stub", "/stubs/canvas/termsSample.txt");
+			fr = retrieveTestFile(url, fr, "Courses call stub 1", "/stubs/canvas/termsSample.txt");
```

Comment:
Isn't this the 'Terms call sub'?  The messages should identify themselves clearly.